README.md

```diff
@@ -229,6 +221,11 @@ The test could be run via `neotest-busted` itself but I decided to use plenary
 instead to use another test runner so that bugs in `neotest-busted` won't affect
 its own tests.
 
+#### Q: Can I use `neotest-busted` with pure lua libraries (no neovim)?
+
+This is not currently possible so please open an issue if you would like this
```

Comment:
See #58 . Let me know if I can help. I would be happy to contribute. But would want some pointers as to how you would want this designed (in terms of config etc), before I invest some time 😄